common/src/main/java/org/apache/sedona/common/raster/RasterConstructors.java

```diff
@@ -26,6 +26,7 @@
 import org.geotools.gce.geotiff.GeoTiffReader;
 import org.geotools.geometry.Envelope2D;
 import org.geotools.geometry.jts.JTS;
```

Comment:
Why does this PR involve changes related to AsRaster? Is this a API breaking change?

If this change is not related to zonal statistics. It must be placed in a separate PR.